crates/relayer/src/light_client/tendermint.rs

```diff
@@ -207,13 +207,18 @@ impl super::LightClient<CosmosSdkChain> for LightClient {
                 challenging_block,
             })) => {
                 warn!("misbehavior detected, reporting evidence to RPC witness node and primary chain");
+                debug!("evidence: {evidence:#?}");
+                debug!("challenging block: {challenging_block:#?}");
+
+                warn!("waiting 5 seconds before reporting evidence to RPC witness node");
```

Comment:
This seems to be needed because without it I sometimes get the following error when reporting the evidence:

> failed to add evidence: Invalid evidence: don't have commit at height #29

I wonder if we can do something smarter, though?

cc @ancazamfir